database/config/config.js

```diff
@@ -1,28 +1,28 @@
-const keys = require("../../utils/keys");
+const keys = require('../../utils/keys');
 
 const config = {
   development: {
-    username: "root",
+    username: 'root',
     password: null,
     database: keys.DB_NAME,
     host: keys.DB_HOST,
```

Comment:
Apply the same logic to the rest of the different environments. My suggestion is to import the env module and access directly the private information here instead of using the keys file.